modules/base/admin/module.py

```diff
@@ -204,8 +204,21 @@ async def repository_install(self, ctx, url: str, branch: Optional[str] = None):
     @commands.max_concurrency(1, per=commands.BucketType.default, wait=False)
     @commands.check(check.acl)
     @repository.command(name="update", aliases=["fetch", "pull"])
-    async def repository_update(self, ctx, name: str):
-        """Update module repository."""
+    async def repository_update(self, ctx, name: str, option: str = None):
```

Comment:
Well, the type hint for `option` is wrong, it should be `Optional[str]`.

Reply:
Done.